test/e2e/static_authorizer.go

```diff
@@ -39,19 +39,48 @@ func testStaticAuthorizer(client kubernetes.Interface) kubetest.TestSuite {
 				given: kubetest.Actions(
 					kubetest.CreatedManifests(
 						client,
+						"authz-static/serviceAccount-krp.yaml",
+						"authz-static/serviceAccount-client-static.yaml",
+						"authz-static/clusterRole-krp.yaml",
+						"authz-static/clusterRoleBinding-krp.yaml",
 						"authz-static/configmap.yaml",
-						"authz-static/clusterRole.yaml",
-						"authz-static/clusterRoleBinding.yaml",
 						"authz-static/deployment.yaml",
 						"authz-static/service.yaml",
-						"authz-static/serviceAccount.yaml",
 					),
 				),
 				check: kubetest.Actions(
 					kubetest.ClientSucceeds(
 						client,
 						fmt.Sprintf(command, "/metrics"),
-						nil,
+						&kubetest.RunOptions{
+							ServiceAccount: "client-with-static-match",
+						},
+					),
+				),
+			},
+			{
+				name: "static/forbidden -> rbac/granted",
```

Comment:
We need good test names given the test framework makes the tests super cryptic on their own. I am having a hard time understanding what this might mean.